### PR TITLE
feat(Alert): add Alert component

### PR DESCRIPTION
**Pull request by wzpo**
feat(alert): Alert Component

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/collab-ui/collab-ui-angular/blob/master/CONTRIBUTING.md#commit-message-format
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")
- [ ] Bugfix
- [x] Feature
- [ ] Documentation update
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)
Add Alert component

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -53,6 +53,7 @@
     "karma-coverage-istanbul-reporter": "~2.0.1",
     "karma-jasmine": "~1.1.2",
     "karma-jasmine-html-reporter": "^0.2.2",
+    "lodash": "^4.17.4",
```
Comment (mohammedzamakhan) [inline package.json]: I would prefer to use `lodash-es` instead for tree shaking

File: src/lib/alert/alert-config.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
+export type AlertPosition = 'top-left' | 'top-center' | 'top-right' | 'bottom-left' | 'bottom-center' | 'bottom-right';
+export type AlertType = 'info' | 'success' | 'warning' | 'error';
+
+/**
+ * Configuration used when opening an alert.
+ */
+export class AlertConfig {
```
Comment (mohammedzamakhan) [inline src/lib/alert/alert-config.ts]: Why not just `interface`?

File: package.json (hunk 1 above)
Comment (wzpo) [reply package.json]: changed to `lodash-es`

File: src/lib/alert/alert-config.ts (hunk 2 above)
Comment (wzpo) [reply src/lib/alert/alert-config.ts]: Because an interface cannot have an initializer.

File: src/docs/app/playground/playground.component.html [hunk 3]
```diff
@@ -1 +1,19 @@
 <p>Your playground code goes here</p>
+
```
Comment (elenagiere) [inline src/docs/app/playground/playground.component.html]: Just a reminder to remove any changes made in playground.component.html and in playground.component.ts

File: src/lib/alert/alert-container.component.html [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+  <cui-alert
+    [key]="alert.key"
+    [ariaLabel]="alert.ariaLabel"
+    [type]="alert.type"
+    [title]="alert.title"
+    [message]="alert.message"
+    [closable]="alert.closable"
+    (dismissed)="onHide($event)"
+  ></cui-alert>
+</ng-container>
```
Comment (elenagiere) [inline src/lib/alert/alert-container.component.html]: very nitpicky, but try to add an empty line at the end of files.

File: src/docs/app/playground/playground.component.html (hunk 3 above)
Comment (wzpo) [reply src/docs/app/playground/playground.component.html]: removed

File: src/lib/alert/alert-container.component.html (hunk 4 above)
Comment (wzpo) [reply src/lib/alert/alert-container.component.html]: Empty line added in two template files.

File: src/lib/alert/alert-config.ts (hunk 2 above)
Comment (mohammedzamakhan) [reply src/lib/alert/alert-config.ts]: Makes sense 👍

File: src/lib/alert/alert-container.component.html (hunk 4 above)
Comment (pauljeter) [reply src/lib/alert/alert-container.component.html]: This should be done automatically by the .editorconfig, which code editor are you using?

Comment (wzpo) [reply src/lib/alert/alert-container.component.html]: VSCode. The .editorconfig does not show the error in **.html** files.

**Pull request merged**